src/ngraph/frontend/onnx_import/op/hardmax.cpp

```diff
@@ -34,11 +34,11 @@ namespace ngraph
                 NodeVector hardmax(const Node& node)
                 {
                     const auto input = node.get_ng_inputs().at(0);
-                    const auto& input_shape = input->get_shape();
+                    const auto& input_shape = input->get_output_partial_shape(0);
```

Comment:
Consider renaming variable to `input_pshape`.

Reply:
According to arrangements with @postrational we should use convention convention `*_shape`